### DISCUSSION TITLE
Workflow stopped working after upgrade

### DISCUSSION THREAD

**Discussion by tanken** [argoproj/argo-workflows#10866] [Q&A]
Am using bitnami/helm chart to perform upgrades from chart 2.4.2 to 3.0.0, the only change was that workflow controller was upgraded from 3.3.8-scratch-r5 to 3.3.9-scratch-r1.

Everything looks fine until I ran a workflow, it got stucked on the first step in running step indefinitely even though the logs should that it has completed. I do not see anything meaningful from the k8s logs except for one that says "node ip-xx.compute.internal cannot be removed: <pod> is not replicated", am not sure if this could be the issue?

Anyone has any suggestions?

**Comment by tanken** [comment]
kubelet.go:2145] "SyncLoop (PLEG): event for pod" pod="abc/123" event=&{ID:... Type:ContainerDied Data:...}
from kubelet.service
Also saw this

**Comment by terrytangyuan** [comment]
Try the latest releases?

**Reply by tanken** [reply]
Unfortunately I cant update to the latest release just yet bcos im limited by the postgres version which is currently 14.5, the latest one is 15.0
